datapackage.json

```diff
@@ -0,0 +1,217 @@
+{
+  "name": "Unicode-Codes",
```

Comment:
@solo11 this should be lower-case, here is the spec: https://specs.frictionlessdata.io/data-package/#metadata